### PR TITLE
Close #19, close #7, close #20: User profiles, welcome screen, handler system

### LINKED ISSUE #7: Set up command handling system

### LINKED ISSUE #19: Get User profile

### LINKED ISSUE #20: Set Welcome screen to init users

### PR DESCRIPTION
**Pull request by bobheadxi**
### Related Issues
Closes #19 - get User profile from Facebook
Closes #20 - welcome screen
Closes #7 - command handling system

### Description
- 🔍 `facebook.API` function to get User profile details (name, timezone, etc)
- 📫 Postbacks are now handled by `facebook.API`
- 👋  Bot now has a welcome screen when a user first interacts with it (delete your old conversation with Calories to see it), and pressing the new "Get Started" button automatically adds you to the database
- 💬 Flexible command and postback handling system in Bot
- 🚧 Other minor fixes (e.g. config val checks) and refactors (e.g. `sender.go` -> `action.go`)

### Todos
none for now 🎉

### CODE REVIEW

**Review by bwdmonkey — DISMISSED** [review]
nice work 👍

**Review by zhulia96 — DISMISSED** [review]
Too clean ✨

**Comment by ayaMakh** [comment]
Noticed some typos

File: bot/bot.go [hunk 1]
```diff
@@ -24,18 +36,64 @@ func New(api *facebook.API, sv *server.Server) *Bot {
...
-	b.api.SendTextMessage(sender.ID, "Hello!")
+// MessageHandler : Handles messages
+func (b *Bot) MessageHandler(event facebook.Event, sender facebook.Sender, msg facebook.ReceivedMessage) {
+	context := &Context{
+		facebookID: sender.ID,
+		timestamp:  event.Time,
+		content:    msg.Text,
+	}
+	// TODO : make command recognition smarter
+	handler, found := b.commands[msg.Text]
```
Comment (ayaMakh) [inline bot/bot.go]: please ignore case

**Review by ayaMakh — APPROVED** [review]
C L E A N

**Pull request merged**